Windows build is broken

1. The windows build is broken due to syslog changes:
```
GOOS=windows GOARCH=386 go build -ldflags="-s -w -X main.gitTag=v2.0.1-180703 -X github.com/dedis/onet.gitTag=v2.0.1-180703" -o conode-v2.0.1-180703/conode.exe
# github.com/dedis/onet/log
../../onet/log/loggers.go:116:10: undefined: syslog.Writer
```

2. This was not discovered until today because making a binary distribution is not part of Travis/Jenkins. We need "make -C conode bindist" in .travis.yaml someplace.

--- a/conode/Makefile
+++ b/conode/Makefile
@@ -68,7 +68,7 @@ bindist: exe/conode.Linux.x86_64
 	cp exe/conode.Linux.x86_64 $(OUTPUT_DIR)
 	GOOS=linux  GOARCH=arm go build $(flags) -o $(OUTPUT_DIR)/conode.Linux.armv7l
 	GOOS=freebsd GOARCH=amd64 go build $(flags) -o $(OUTPUT_DIR)/conode.FreeBSD.amd64
-	GOOS=windows GOARCH=386 go build $(flags) -o $(OUTPUT_DIR)/conode.exe
+	GOOS=windows GOARCH=amd64 go build $(flags) -o $(OUTPUT_DIR)/conode.exe
 	echo "#!/bin/sh" > $(OUTPUT_DIR)/conode
 	echo "./conode.\`uname -s\`.\`uname -m\` \$$*" >> $(OUTPUT_DIR)/conode
 	chmod +x $(OUTPUT_DIR)/conode

